presto-parser/src/test/java/com/facebook/presto/sql/parser/TestSqlParser.java

```diff
@@ -1541,6 +1541,14 @@ public void testExists()
                                         new ExistsPredicate(simpleQuery(selectList(new LongLiteral("2"))))))));
     }
 
+    @Test
+    public void testFormatter()
+    {
+        Statement parsed = SQL_PARSER.createStatement("prepare p from select * from (select * from T) \"A B\"");
```

Comment:
For this type of round-trip smoke test, you can use `printStatement()` in `TestStatementBuilder`. It calls `assertFormattedSql()` to do the round trip test.

I'll move this to there.